xbmc/filesystem/Directory.cpp

```diff
@@ -390,4 +390,4 @@ void CDirectory::FilterFileDirectories(CFileItemList &items, const std::string &
         }
     }
   }
-}
+}
```

Comment:
Can't really comment on the code although I will include it in tonight's LE test build, but can we avoid this kind of unnecessary change?